server/api/user.go

```diff
@@ -140,6 +140,26 @@ func GetRepos(c *gin.Context) {
 		return
 	}
 
+	repoIDs := make([]int64, len(activeRepos))
```

Comment:
Would it make sense to use an extended model just for this api endpoint to keep the normal pipeline model lean. Sth like:

```go
type PipelineWithLastPipeline struct {
	model.Pipeline
	LastPipeline *model.Pipeline `json:"last_pipeline"`
}
```

Reply:
What's the (real) benefit?

Reply:
Openapi would imply on all other endpoints that a repo could include a pipeline which is never the case. In addition the model is used for grpc etc and would imply extra properties which are never used / assigned.

Reply:
@anbraten changed